### DISCUSSION TITLE
Vercel .env not working

### DISCUSSION THREAD

**Discussion by FrankSiyli** [neuefische/questions#580] [Web]
Hello,

NEXT.js App

Failure: My "session" objects from the DB don't get fetched in Vercel deployment.

I created a MongoDB for my App.
From the DB I fetch my "session" objects to display them in my App.
The key and value pair is hidden in the .env and the .env is added to the .gitignore.

With VSCode everything is fine on my localhosts. My fetched sessions are displayed. I've checked it in different browsers, no console.log errors.

I've added the key and value pair in Vercel via both ways. 
First way was via the Vercel GUI on the website and the second way was via the Vercel CLI in iTerm2.

I've redeployed the project deployment and the branch deployment after setting the key several times. 

[Vercel branch deployment](https://capstone-triathlon-app-git-us6-franksiyli.vercel.app) with the failure.

[Repo on Github](https://github.com/FrankSiyli/capstone_triathlon_app).

Thanks for any advice and best regards Frank.

**Comment by JessicaLoers** [comment]
Hi @FrankSiyli ✌️
I have a wild guess. 
From your error description above, I could imagine it's because you need to change your `IP Access List entries` at MongoDB Atlas from your current IP Adress to `ALLOW ACCESS FROM ANYWHERE`.

**Please have a look at steps 10 and 11 in the corresponding [MongoDB Atlas handout](https://github.com/neuefische/cgn-web-23-1/blob/main/sessions/backend-mongodb-atlas/backend-mongodb-atlas.md).**

Otherwise please take a look at the` runtime logs` of your Vercel deployment. **Are there any error messages there?**
If so, please provide us with it 🙏
